schemas/validation_tests/ArrayExperiment-invalid-4-wrong-filename.json

```diff
@@ -0,0 +1,90 @@
+{
+    "object_id": {
+        "alias" : "MyArrayExperiment_13", 
+        "center_name" : "EBI-TEST"
+    },
+    "title": "Testing ArrayExperiment title",
+    "description": "Broad description of the testing ArrayExperiment",
+    "technology": {
+        "technology_id": "One-colour microarray",
+        "technology_curie": "EFO:0010939"
+    },
+    "array_label": [
+        {
+            "array_label_id": "Cy3 dye",
+            "array_label_curie": "CHEBI:37987"
+        },
+        {
+            "array_label_id": "Biotin",
+            "array_label_curie": "CHEBI:15956"
+        }
+    ],
+    "experimental_design": {
+        "experimental_design_id": "dose response design",
+        "experimental_design_curie": "EFO:0001757"
+    },
+    "array_type": {
+        "array_type_id": "ChIP-chip by array",
+        "array_type_curie": "EFO:0002760"
+    },
+    "assayed_molecule": {
+        "assayed_molecule_id": "DNA assay",
+        "assayed_molecule_curie": "EFO:0001456"
+    },
+    "sample_number": 12,
+    "ADF_files": [
```

Comment:
```suggestion
    "adf_files": [
```

Reply:
I suggest having a rule that all attributes are lower_camel_case, including acronyms. For example, you use `id` and not `ID` .